src/Sarif/Baseline/V2/WhatComponent.cs

```diff
@@ -14,10 +14,10 @@ internal class WhatComponent : IEquatable<WhatComponent>
 
         public WhatComponent(string category, string propertySet, string propertyName, string propertyValue)
         {
-            this.Category = category;
-            this.PropertySet = propertySet;
-            this.PropertyName = propertyName;
-            this.PropertyValue = propertyValue;
+            this.Category = category ?? "";
```

Comment:
Yes. I prefer "" in the null coalescing expressions because it's drastically shorter and then you can usually read what it is doing with the expression.

I still don't actually know why we recommended string.Empty. I'm pretty sure "" will continue to mean string.Empty forever.